scene/debugger/scene_debugger.cpp

```diff
@@ -296,6 +298,37 @@ Error SceneDebugger::parse_message(void *p_user, const String &p_msg, const Arra
 			return ERR_SKIP;
 		}
 
+#ifdef TOOLS_ENABLED
+	} else if (p_msg.begins_with("editor_setting_")) {
+		ERR_FAIL_COND_V(p_args.size() < 2, ERR_INVALID_DATA);
+
+		const String setting = p_args[0];
+		Variant value;
+
+		if (p_msg == "editor_setting_value") {
+			value = p_args[1];
+		} else if (p_msg == "editor_setting_shortcut") {
+			VariantParser::StreamString ss;
+			ss.s = p_args[1];
+
+			String errs;
+			int line;
+			VariantParser::parse(&ss, value, errs, line);
```

Comment:
This cause unchecked remote code execution.

Reply:
I just realized this is the same as done in #47744 which effectively allow code execution from an environment variable :(